feat/#52 observe and update the input

arodriguezl9302 opened the pull request:
#### ATTENTION

PLEASE NOTE THAT TO AVOID PROBLEMS RUNNING THIS PR, YOU NEED TO HAVE PR#50 RUNNING

### Description

Changed header styles, added back button, fonts, text field and button styles

fixes #52 

#### Screenshots

![image](https://user-images.githubusercontent.com/24697827/155918051-87ea19b3-17d2-4e71-be86-f29247420dd6.png)


#### Type of change


New feature (non-breaking change which adds functionality)

### How to test it

If applicable, describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

1. Go to '/check'


### Checklist:

- [x] Change header background color with rounded corners according to design color #9DBDFD
- [x] Place back button aligned to the top left according to design
- [x] Change title and subtitle text styles in header type: regular and bold popins as appropriate color #002740
- [x] Change title styles in bold uppercase popins font color #005D9D
- [x] Change size and color of text field, background color #C4C4C4 23%, border #5F6769 to 1px, font used regular popins, all fields capitalized.
- [x] Change button color color #005D9D popins medium uppercase font.

heliomar-pena commented:
I will wait to #70 was merged to check this, to avoid conflicts

angelik0828 commented:
It looks good, I have some suggestions, the button is very small, it should be almost the size of the text field, the ticket image does not go in the layout, you need to change or remove it, please check the margins. that there in the word "hello" at the bottom, attached image with suggestion.

![image](https://user-images.githubusercontent.com/4030477/156656772-15585dea-adf7-4be3-b0f9-ddbb640ae564.png)

![image](https://user-images.githubusercontent.com/4030477/156656589-afc94d89-b1b3-4a0f-bd94-5d2e2ec038af.png)

heliomar-pena commented on src/components/Input.jsx:
```diff
@@ -18,7 +18,7 @@ const Input = styled.input`
       bgColor === 'gray' &&
       css`
-        background-color: var(--light-gray);
+       //background-color: var(--light-gray);
```
Please remove comment

Katsu08 reviewed (changes requested):
Please remove comments

arodriguezl9302 commented:
@Katsu08 @angelik0828

I changed the format of the title and subtitle, removed the notes icon and made the header a bit bigger 

![image](https://user-images.githubusercontent.com/24697827/156695145-344be26c-94cb-4057-9919-07530e41deb0.png)

heliomar-pena commented on src/pages/Chat/Chat.styled.jsx:
```diff
@@ -22,6 +22,7 @@ export const HeaderChat = styled.div`
 
 export const HeaderChatImgContainer = styled.div`
   text-align: right;
+  margin-left: -80px;
```
Please do not use negative margin-left with so big numbers, just use it for little adjustment of fewer than ten pixels. Please check here how the image is positioned correctly without using margin-left: 80px https://github.com/Azordev/maps-project/pull/72/commits/9304213eb9507aaf997f3441eab11a4b94ac0fb8

Katsu08 reviewed (approved):
Good work!

angelik0828 commented:
Thanks for the changes, I see that the button is still in a smaller size, we have talked and it will be handled with another problem since the component used is on another screen.

Katsu08 reviewed (approved):
Good work!

heliomar-pena commented on src/pages/Check/Check.styled.jsx:
```diff
@@ -2,18 +2,27 @@ import styled from '@emotion/styled';
+`
+
+export const SubtitleContainer = styled.div`
+  margin-top: -40px;
```
Can you check if exists a better solution to position subtitle container?

Katsu08 reviewed (changes requested):
Good job!. I have left you a comment about use of negative margin

Katsu08 reviewed (approved):
Good work man!
